cpp/log/sqlite_db-inl.h

```diff
@@ -21,7 +23,14 @@ DEFINE_bool(sqlite_synchronous_off, false,
 namespace {
 
 
+static cert_trans::Latency<std::chrono::milliseconds, std::string>
+    latency_by_op_ms("database_latency_by_operation_ms", "operation",
```

Comment:
Yeah, that was kinda deliberate; I figured that the server is only going to use one of these, and the operations are the same for both - this would save you from having to change all your monitoring if you switched between them.

Really this should be done at the `database.cc` level and rename all the virtual methods, but for now I'll just rename the metric names and we'll do the `database.cc` thing as/when.